src/pages/stacks-blockchain/running-testnet-node.md

```diff
@@ -13,9 +13,15 @@ images:
 
 ## Introduction
 
-The Stacks 2.0 testnet is currently in development. As part of the testnet, you can run a node and connect it to a public network. This guide will walk you through downloading and running your own node in the testnet network.
+-> Note: The Stacks 2.0 testnet is currently in development. As part of the testnet, you can run a node and connect it to a public network.
 
-### Prerequisites
+This tutorial will walk you through the following steps:
+
+- Download and install the node
+- Running the node
```

Comment:
[nit] Consistent tense: "Running" and "Mining" vs. "Download and install" (so if we want to be consistent with latter, we might say "Run the node", "Mine Stacks token")